src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1613,6 +1613,21 @@ public void setResourcePack(@NotNull String url, byte[] hash)
 		throw new UnimplementedOperationException();
 	}
 
+	@Override
+	public void setResourcePack(@NotNull String url, @Nullable byte[] hash, @Nullable String prompt) {
+
+	}
+
+	@Override
+	public void setResourcePack(@NotNull String url, @Nullable byte[] hash, boolean force) {
+
+	}
+
+	@Override
+	public void setResourcePack(@NotNull String url, @Nullable byte[] hash, @Nullable String prompt, boolean force) {
+
+	}
```

Comment:
Invalid implementations, please follow our specifications and throw an `UnimplementedOperationException` for non-implemented method stubs.